src/components/DisclaimerLink/index.tsx

```diff
@@ -0,0 +1,77 @@
+import type { ReactElement } from 'react'
+import { Box, Button, Divider, Paper, Stack, Typography } from '@mui/material'
+import css from './styles.module.css'
+import { InfoOutlined } from '@mui/icons-material'
+import { SAFE_TERMS_AND_CONDITIONS_URL } from '@/config/constants'
+import { ExternalLink } from '../ExternalLink'
+
+export const DisclaimerLink = ({
+  title,
+  buttonText,
+  href,
+  onCancel,
+  onAccept,
+}: {
+  title: string
+  buttonText?: string
+  href: string
+  onCancel: () => void
+  onAccept: () => void
+}): ReactElement => {
+  return (
+    <div className={css.container}>
+      <Paper sx={{ p: 3 }}>
+        <Stack padding="var(--space-3)" gap={2} display="flex" alignItems="center" mb={3}>
+          <Box>
+            <InfoOutlined />
+          </Box>
+          <Typography variant="h3" fontWeight={700}>
+            {title}
+          </Typography>
+          <Typography variant="body2" mb={1}>
+            <b>
+              Safe{'{'}Pass{'}'}Partner third party app.{' '}
+            </b>{' '}
+            <br />
+            By accessing this third party applications provided by a partner of Safe
+            {'{'}Pass{'}'} Program (&quot;Partner&quot; and/or &quot;Safe{'{'}Pass{'}'}Partner&quot;) you acknowledge
+            and agree that Safe Ecosystem Foundation and/or Safe{'{'}Wallet{'}'} do not own, control, maintain or audit
```

Comment:
```suggestion
            and agree that Safe Ecosystem Foundation and/or Safe{'{'}Wallet{'}'} do not own, control, maintain, or audit
```

The written style guide on notion says we should use the oxford comma: https://www.notion.so/safe-global/Written-Style-Guide-281a407dec424c93b23aa51eb9e6ac0c